components/kyma-environment-broker/internal/storage/encrypt.go

```diff
@@ -60,36 +60,12 @@ func (e *Encrypter) Decrypt(obj []byte) ([]byte, error) {
 }
 
 func (e *Encrypter) EncryptSMCreds(pp *internal.ProvisioningParameters) error {
-	if pp.ErsContext.ServiceManager == nil && pp.ErsContext.SMOperatorCredentials == nil {
+	if pp.ErsContext.SMOperatorCredentials == nil {
 		return nil
 	}
 	var err error
 	encrypted := internal.ERSContext{}
-	if pp.ErsContext.ServiceManager != nil {
-		creds := pp.ErsContext.ServiceManager.Credentials.BasicAuth
-		var username, password []byte
-		if creds.Username != "" {
-			username, err = e.Encrypt([]byte(creds.Username))
-			if err != nil {
-				return errors.Wrap(err, "while encrypting username")
-			}
-		}
-		if creds.Password != "" {
-			password, err = e.Encrypt([]byte(creds.Password))
-			if err != nil {
-				return errors.Wrap(err, "while encrypting password")
-			}
-		}
-		encrypted.ServiceManager = &internal.ServiceManagerEntryDTO{
-			URL: pp.ErsContext.ServiceManager.URL,
-			Credentials: internal.ServiceManagerCredentials{
-				BasicAuth: internal.ServiceManagerBasicAuth{
-					Password: string(password),
-					Username: string(username),
-				},
-			},
-		}
-	}
+
 	if pp.ErsContext.SMOperatorCredentials != nil {
```

Comment:
nit: also here I think we can drop this due to
https://github.com/kyma-project/control-plane/blob/b5fd6f562b159bce522b9f83a3d7e9216ca0e493/components/kyma-environment-broker/internal/storage/encrypt.go#L63-L65

Reply:
applied